js/tenPin.js

```diff
@@ -0,0 +1,122 @@
+var TenPin = function(frame) {
+  this.newFrame = frame
+  this.finalFrame = false;
+  this.maxFrame = 10;
+  this.currentFrame = 0;
+  this.framesCreated = [];
+  this.createFrame(this.newFrame);
+};
```

Comment:
You are defining real objects here - spies not really required as you can test the objects directly